src/Pages/Profile/MyAccount.tsx

```diff
@@ -117,29 +117,28 @@ const MyAccount: React.FC = () => {
                 return false; // Prevent default upload behavior
               }}
             >
-              {' '}
               {imageUrl ? (
-                <img src={imageUrl} alt="avatar" style={{ width: '100%' }} />
+                <img src={imageUrl} alt="avatar" className="profile-image" />
               ) : (
                 uploadButton
-              )}
+              )}{' '}
             </Upload>
           </Form.Item>
-          <Form.Item
-            label={
-              <label className="item-form-label">{t('accountPage.name')}</label>
-            }
-          >
-            <Input onChange={onNameChange} value={name || ''} />
+          <Form.Item>
+            <Input
+              onChange={onNameChange}
+              value={name || ''}
+              className="form-input"
+              placeholder={t('accountPage.name') || 'Your name'}
+            />
           </Form.Item>
-          <Form.Item
-            label={
-              <label className="item-form-label">
-                {t('accountPage.emailAddress')}
-              </label>
-            }
-          >
-            <Input onChange={onEmailChange} value={email || ''} />
+          <Form.Item>
+            <Input
+              onChange={onEmailChange}
+              value={email || ''}
+              className="form-input"
+              placeholder={t('accountPage.name') || 'Your name'}
```

Comment:
This should be:
`placeholder={t('accountPage.emailAddress') || 'Your email address'}`